msvc/dfhack-test

```diff
@@ -1,21 +1,11 @@
-#!/bin/bash -e
+#!/bin/bash
```

Comment:
removing -e means we won't get exit statuses from commands that fail

you might want to add `status=$?` after the important command and then `exit $status` at the end

Reply:
Ah, ok. Sorry, I'll fix that. The bash man page didn't really explain what it was for (just that it was inherited from the parent shell) and it seemed to cause the script to have no effect after the first time I ran it. I'll add in a status return when I get the chance

Reply:
`-e` means that if any command returns non-zero, the bash shell immediately exits with that code

Reply:
Ah, ok, I think I understand now. I bet one of the wine commands returns zero the first time it is run and non-zero thereafter. It was very strange to have the scripts exiting with zero output.